grammar.y

```diff
@@ -972,7 +986,35 @@ create_table_stmt:
     if len($5) > MaxAllowedColumns {
       yylex.(*Lexer).AddError(&ErrTooManyColumns{ColumnCount: len($5), MaxAllowed: MaxAllowedColumns})
     }
-    $$ = &CreateTable{Table: $3, Columns: $5, Constraints: $6}
+
+    // We have to replace a primary key table constraint with an equivalent column constraint primary key,
+    // so we can add the autoincrement flag, as part of the rules of the Tableland Protocol.
+    // 
+    // That happens because a primary key table constraint that references a single INTEGER column
+    // would be an alias to rowid. For cases where a column becomes an alias to rowid we want to force the AUTOINCREMENT.
+    // 
+    // The exception to the above rule is when a table constraint primary key has order DESC. In that case, we replace with an 
+    // equivalent column constrain without forcing AUTOINCREMENT.
```

Comment:
Nit: maybe for completeness append: `... forcing AUTOINCREMENT and avoiding being interpreted as an alias`?